python/pyhail/dataset.py

```diff
@@ -689,6 +689,70 @@ def linreg(self, y, covariates="", root="va.linreg", minac=1, minaf=0.0):
         pargs = ['linreg', '-y', y, '-c', covariates, '-r', root, '--mac', str(minac), '--maf', str(minaf)]
         return self.hc.run_command(self, pargs)
 
+    def lmmreg(self, y, kinshipfilter, covariates=None, assocfilter=None, noassoc=None,
+               useml=None, globalroot=None, varoot=None, delta=None,
+               sparsitythreshold=None, forceblock=None, forcegrammian=None):
```

Comment:
Changed None to False for Booleans, and empty string for covariates.  I'll refactor all regression methods to list of string in later PR